packages/synapse-react-client/src/lib/containers/StorybookComponentWrapper.tsx

```diff
@@ -134,20 +135,21 @@ export function StorybookComponentWrapper(props: {
     [accessToken],
   )
 
+  const theme = createTheme(defaultMuiThemeOptions, {
+    palette: paletteMap[storybookContext.globals.palette],
+  })
```

Comment:
Small change, create the theme in Storybook the same way we create it in the apps.